scripts/composer_update.sh

```diff
@@ -0,0 +1 @@
+docker run --rm --interactive --tty --volume $PWD:/app composer update --ignore-platform-reqs --no-scripts
```

Comment:
Should add the [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) line to this.
Also, did you remember to `chmod +x` before committing it? I can't tell from the web interface.